components/sup/src/manager/service/mod.rs

```diff
@@ -660,6 +662,7 @@ impl Service {
     }
 
     /// Write service files from gossip data to disk.
+    /// For the location, check `svc_file_path()`.
```

Comment:
Wasn't sure how to create a symlink for this. Also, the function is private, so I'm not sure if there's a point.

Reply:
I guess this is a note to yourself, because I don't understand what it is about. Not enough context.

Reply:
The `svc_file_path` function could be rendered as a link, since `///` comments are Markdown. I just don't know what the target should be, as it seems to be relative to something.

Reply:
Fixed this.